lib/payloadAction.ts

```diff
@@ -74,6 +74,17 @@ export const  handlePayloadActions = async (app: IApp, read: IRead,  modify: IMo
 
                         // Start blackout window
                         if (params.continue_blackout) {
+                            const continueBlackoutTask = {
+                                id: JobName.QUEUE_WINDOW_SCHEDULER,
+                                when: `120 seconds`,
+                                data: {
+                                    rid,
+                                    sessionId: rid,
+                                    jobName: JobName.EVENT_SCHEDULER,
+                                },
+                            };
```

Comment:
Here, the `id` is `JobName.QUEUE_WINDOW_SCHEDULER,` but the `jobName` is `JobName.EVENT_SCHEDULER`. It should be same right?